### DISCUSSION TITLE
Allow filtering Search Console terms by entry page.

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1268] [Anything Else]
It's currently possible to see this data from Google Search Console itself, but would be nice if setting two filters:

- Source is Google
- Entry page is `/some_page`

would allow seeing the search terms that drive users to your site via `/some_page` directly on your Plausible dashboard.

**Update:** Search Console filtering is here! You can now filter your dashboard by page, country or device and see the Search Console keyword terms for that segment of your audience only. We now also display the number of impressions, CTR and position for each search term.

**Comment by jobee** [comment]
Hi there, I am surprised that this topic does not have much response.

Showing all keywords for a specific time range, not considering current filters like Country, Entry page, Devices is not very helpful most of the time. Also filtering by a specific keyword would be something users benefit from.

Is this on your roadmap or a limitation in the SearchConsole API right now?

**Comment by metmarkosaric** [comment]
**Update:** Search Console filtering is here! You can now filter your dashboard by page, country or device and see the Search Console keyword terms for that segment of your audience only. We now also display the number of impressions, CTR and position for each search term.
